src/js/common/components/RequiredLoginView.jsx

```diff
@@ -30,18 +30,16 @@ class RequiredLoginView extends React.Component {
     const userServices = this.props.userProfile.services;
     if (userServices) {
       this.setState(
-        {
-          isServiceAvailableForUse: userServices.includes(requiredApp),
-        }
+        { isServiceAvailableForUse: userServices.includes(requiredApp) }
       );
     }
   }
 
   render() {
     let view;
 
-    const loginComponent = <LoginPrompt loginUrl={this.props.loginUrl}/>;
-    const verifyComponent = <VerifyPrompt/>;
+    const loginComponent = <LoginPrompt loginUrl={this.props.loginUrl.first}/>;
+    const verifyComponent = <VerifyPrompt verifyUrl={this.props.loginUrl.third}/>;
```

Comment:
Yeah--those fields were already there, though I don't think they're that helpful vs. just having 2 separate props. Unless...maybe there's a way to fetch both URLs at once, in which case it would be helpful to be able to stuff them into the same object all at once. I will poke around the ID.me logs and/or ask on slack if anyone knows